humidity_sensor/humidity_sensor.ino

```diff
@@ -22,13 +22,8 @@ const int led2 = 12;
 const int led3 = 11;
 const int BUTTON = 7;
 
-int buttonState = 0;
-int lastButtonState = 0;
-int state = 0;
-
-float humidity;
-float temperature;
-float pressure;
+int buttonState,lastButtonState,state = 0;
+float humidity,temperature,pressure = 0.00; 
```

Comment:
I'd initialize them to MIN_FLOAT_VALUE and add a comment explaining the will be set immediately